src/docfx/lib/HtmlUtility.cs

```diff
@@ -104,6 +104,19 @@ internal static class HtmlUtility
             { "strike", null },
         };
 
+        private static string[] inlineTags = new[]
```

Comment:
See `3.` section of [this coding guideline](https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/coding-style.md) we use:
```suggestion
        private static string[] s_inlineTags = new[]
```